src/Microsoft.CodeQuality.Analyzers/UnitTests/ApiDesignGuidelines/UriParametersShouldNotBeStringsTests.Fixer.cs

```diff
@@ -66,33 +47,44 @@ public static void Method(Uri url)
         }
 
         [Fact]
-        public void CA1054MultipleWarningWithUrl()
+        public async Task CA1054MultipleWarningWithUrl()
         {
             var code = @"
 using System;
 
 public class A
 {
-    public static void Method(string url, string url2) { }
+    public static void Method(string [|url|], string [|url2|]) { }
 }
 ";
+
+#if false // Not currently supported by the new test harness
```

Comment:
❓ @genlu  @mavasani Should we remove this test case, or file a bug on dotnet/roslyn-sdk to support it?

Reply:
Could you please help me understand the problem?

Reply:
The test harness iteratively applies fixes to reach a stable state. It does not support applying one fix and stopping at an intermediate state that still has diagnostics. It's a hard problem because it interacts in strange ways with Fix All test support.

Reply:
As best I can tell, we can:

1. Accept this as a limitation of the new test harness which isn't particularly important to address, and delete this test case
1. Treat this as a _non-blocking_ limitation of the test harness, and file a dotnet/roslyn-sdk issue to support it in the test harness and a dotnet/roslyn-analyzers issue to use it when available
1. Treat this as a _blocking_ limitation of the test harness, and review this change again only after the underlying limitation has been corrected and this PR updated to use it

Reply:
I think it is fine to split this into 2 tests, one with FixAll validation where we multiple diagnostics/fixes and other with a single diagnostic/fix where FixAll result is identical to simple code fix application. I am not sure if the original test's intent was to intentionally verify intermediate state.

Reply:
1 or 2 seems fine to me.

Reply:
@mavasani I went with option 1. Other tests in the file already cover the single-fix case, so I just removed the extra code.